addons/mail/models/discuss/discuss_channel.py

```diff
@@ -600,18 +600,7 @@ def _notify_thread(self, message, msg_vals=False, **kwargs):
         message_format = message.message_format()[0]
         if "temporary_id" in self.env.context:
             message_format["temporary_id"] = self.env.context["temporary_id"]
-        # Last interest and is_pinned are updated for a channel when posting a message.
-        # So a notification is needed to update UI, and it should come before the
-        # notification of the message itself to ensure the channel automatically opens.
-        payload = {
-            "Thread": {
-                "id": self.id,
-                "last_interest_dt": fields.Datetime.now(),
-                "model": "discuss.channel",
-            },
-        }
         bus_notifications = [
-            (self, "mail.record/insert", payload),
```

Comment:
You should also adapt the mock server, otherwise the test coverage of unit tests do not match this new behaviour

Reply:
yes, as talked with @seb-odoo yesterday, this was an urgent bug fix and the adapt on the mock server will be just right after the fix, with a separate PR so that the fix can be merged soon.

Reply:
ok i will change the mock server in the PR

Reply:
done